Client.xcodeproj/project.pbxproj

```diff
@@ -6050,10 +6052,11 @@
 		F84B22211A09122500AAB793 /* Home */ = {
 			isa = PBXGroup;
 			children = (
-				74821F8D1DAD8F1300EEEA72 /* FirefoxHomeHighlightCell.swift */,
-				3BB50E1F1D627539004B33DF /* FirefoxHomeViewController.swift */,
 				2386E4E524F8358E0072EF17 /* DefaultBrowserCard.swift */,
+				CE4C3DC325ED41E90049E9BF /* FeedbackCard.swift */,
```

Comment:
It's a new file, isn't it? I'd say let's put into our Ecosia folder to have less potential conflicts in future merges of the project file and for house-keeping, don't you think?